Add VPC peering between the CyHy and BOD VPCs

jsf9k opened the pull request.

jsf9k commented on ansible/roles/mongo/tasks/main.yml:
```diff
@@ -28,6 +28,13 @@
+- name: Configure mongo service to listen on all interfaces
+  lineinfile:
+    dest: /lib/systemd/system/mongod.service
+    regexp: '^( *bindIp=.*)$'
```
This isn't working.  :angry:   I need to fix this before we merge.

dav3r replied:
There are a couple other changes that I made to the mongo config file (located at /etc/mongod.conf, not /lib/systemd/system/mongod.service), so let's coordinate on those too.

jsf9k commented:
Building up and tearing down is complicated now that we have two Terraform projects (`terraform` and `terraform _bod`) that are interdependent.  Maybe we should move the `terraform_bod` code into the `terraform` directory.

What do you guys think?

dav3r commented:
@jsf9k , I agree that if the `terraform` and `terraform_bod` directories are now interdependent, they should be merged.

jsf9k commented:
@dav3r, I moved the `terraform_bod` code into the `terraform` directory (and renamed and modified some files as necessary).  Let's get those Mongo configuration changes you have in so we can merge this one!
:penguin: :tada: :trumpet: :smiling_imp: :cat2: :koala: :dolphin:
